test/Analyzers.Tests/Wrapper.cs

```diff
@@ -48,10 +48,16 @@ public static void Main()
     static string Usings()
     {
         return $@"
+using Azure.Storage.Blobs;
+using Azure.Storage.Queues;
+using Azure.Data.Tables;
 using System;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Data.SqlClient;
```

Comment:
Do we need these using statements?

Reply:
Yes, we need them so the unit tests can run. The Wrapper methods add those usings to the wrapped classes/methods/etc. For the cases which the usings aren't needed (not all test classes/methods will use them all), the compiler will just discard them.